js/stdlib/ts/ALGO.ts

```diff
@@ -593,17 +593,21 @@ type QueryResult =
 // Event Cache
 // ****************************************************************************
 
-export const chooseMinRoundTxn = (ptxns: any[]) =>
+const chooseMinRoundTxn = (ptxns: any[]) =>
   ptxns.reduce((accum: any, x: any) =>
       (x['confirmed-round'] < accum['confirmed-round']) ? x : accum, ptxns[0])
 
+const chooseMaxRoundTxn = (ptxns: any[]) =>
```

Comment:
These two functions suggest adding `argmax` to our JS utilities --- https://docs.racket-lang.org/reference/pairs.html#%28def._%28%28lib._racket%2Flist..rkt%29._argmax%29%29